components/eamxx/src/diagnostics/atm_backtend.hpp

```diff
@@ -16,7 +16,7 @@ class AtmBackTendDiag : public AtmosphereDiagnostic {
   AtmBackTendDiag(const ekat::Comm &comm, const ekat::ParameterList &params);
 
   // The name of the diagnostic
-  std::string name() const;
+  std::string name() const { return m_name + "_atm_backtend"; }
```

Comment:
@mahf708 I did not want the output yaml files ot change for this var, so instead I changed the name of the output var. I think it's more important to keep old yaml files working than keeping the same output fields names in the nc files (especially since I don't hink the backtend diag was widely used, I hope I'm not wrong on this).